Cargo.lock

```diff
@@ -398,7 +398,7 @@ dependencies = [
  "bitflags 2.5.0",
  "cexpr",
  "clang-sys",
- "itertools 0.12.1",
+ "itertools 0.11.0",
```

Comment:
I have no idea how cargo came to this conclusion to downgrade here in a far build dependency (`bindgen`)

Saw even weirder downgrades in #12459